pkg/controller/servicebindingrequest/olm.go

```diff
@@ -42,7 +42,10 @@ func (o *OLM) listCSVs() ([]unstructured.Unstructured, error) {
 	gvr := olmv1alpha1.SchemeGroupVersion.WithResource(csvResource)
 	resourceClient := o.client.Resource(gvr).Namespace(o.ns)
 	csvs, err := resourceClient.List(metav1.ListOptions{})
-	if err != nil {
+	if err != nil && errors.IsNotFound(err) {
```

Comment:
Thank you! do you think a test could be added this?

Reply:
I added a test case to mock a not found error like a genuine API server. Please Review. Thanks!